ui/pages/Project/components/AnalysisGroupButtons.jsx

```diff
@@ -28,39 +29,37 @@ const FAMILY_FIELDS = [
   { name: FAMILY_FIELD_DESCRIPTION, width: 9, content: 'Description' },
 ]
 
-const parseFamilyGuids = (newValue, previousValue, allValues) => {
-  const parsed = {}
-  if (newValue.parsedData && newValue.uploadedFileId !== (previousValue || {}).uploadedFileId) {
-    parsed.familyIdMap = allValues.allFamilies.reduce(
-      (acc, family) => ({ ...acc, [family.familyId]: family.familyGuid }), {},
-    )
-    parsed.familyGuids = newValue.parsedData.map(row => parsed.familyIdMap[row[0]]).filter(familyGuid => familyGuid)
-  }
-  return parsed
-}
-
-const normalizeFamilyUpload = (newValue, previousValue, allValues) => {
```

Comment:
There are 2 ways to update families in this form, uploading a file or selecting in a table. Previously, we had both fields using the same field name, and a custom `normalize` function that looked at previous values for the field to properly merge the input. Now these are broken in to 2 separate fields, and we use a calculate decorator that causes the familyGuids field to update whenever the uploadedFamilyIds field updates to keep the 2 fields in sync.